Automattic/jetpack
# Better Video Privacy Section Label for the Video Block

Changes the section label to read "Video Privacy" as suggested in pxWta-1kj-p2#comment-5651

#### Changes proposed in this Pull Request:

* Label change

#### Jetpack product discussion

pxWta-1kj-p2#comment-5651

#### Does this pull request change what data or activity we track or use?

no


#### Testing instructions:
* Select a video block. The Video Privacy section label should be "Video Privacy"








* Go to '..'
*
